__tests__/e2e/main.test.ts

```diff
@@ -203,6 +203,44 @@ describe('GitHub Action', () => {
     )
   })
 
+  it('should assign job id 0 to the first created job', async () => {
+    const gitRepo = await createInitializedGitRepo()
+
+    const output = createJob(gitRepo)
+
+    expect(getOutputVariable('job_id', output.toString())).toBe('0')
+  })
+
+  it('should assign job id 1 to the second created job', async () => {
+    const gitRepo = await createInitializedGitRepo()
+
+    const firstOutput = createJob(gitRepo)
+
+    expect(getOutputVariable('job_id', firstOutput.toString())).toBe('0')
+
+    executeAction({
```

Comment:
@yeraydavidrodriguez maybe we can create a wrapper for this case like `createJob(gitRepo)`. In fact, I think we should rename it to `createSampleJob(...)`. And also for the finish invocation.

```
createSampleJob(gitRepo, dummyPayload())
startSampleJob(gitRepo, dummyPayload())
finishSampleJob(gitRepo, dummyPayload())
```

Reply:
Done, much better now